hackathon_site/event/jinja2/event/landing.html

```diff
@@ -12,299 +14,365 @@
+                    </div>
                 </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">12PM</p>
-                    <p>Judging</p>
+            </div>
+            <div class="m12 center">
+                <a href="#" class="btn-large waves-effect waves-light colorBtn" rel="noopener" target="_blank">Full schedule</a>
+            </div>
+        </div>
+    </div>
+    <br /><br /><br />
+
+    <div class="container colorScroll" data-background-color="#003650">
+        <div class="section scrollspy" id="faq">
+            <div class="row">
+                <div class="col s12 m12 center">
+                    <h3 class="center">FAQ</h3>
                 </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">3PM</p>
-                    <p>Closing Ceremonies</p>
+                <div class="col s12 m6">
+                    <ul class="collapsible expandable">
+                        <li>
+                            <div class="collapsible-header">
+                                What is a makeathon?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    A makeathon is like a hackathon but more hands-on. It involves the integration of hardware and software to build a working device that could be used in any application from gaming to solving a real life problem! As such, we require all participants to build some sort of physical device that is at the very least electrically powered or computer programmed.
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                What if I have no experience? Can I still come?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Yes! MakeUofT is for everyone, from beginners to veterans. You will have the opportunity to learn technical skills from industry mentors. Also, if you ever get stuck on your project just ask one of the mentors!
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                Will you be shipping out hardware?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Unfortunately, our team would not be able to ship out any hardware for MakeUofT 2021. But the first 200 people to apply will be able to get 25 dollars reimbursement after submitting a demo with their team!
```

Comment:
"would not" -> "will not"
"first 200 people to apply" -> "first 200 people to be accepted"